images/iqsharp-base/Dockerfile

```diff
@@ -93,7 +93,7 @@ ENV PATH=$PATH:${HOME}/dotnet:${HOME}/.dotnet/tools \
 # Install IQ# and the project templates, using the NuGet packages from the
 # build context.
 ARG IQSHARP_VERSION
-RUN dotnet new -i "Microsoft.Quantum.ProjectTemplates::0.10.1910.2802-beta" && \
+RUN dotnet new -i "Microsoft.Quantum.ProjectTemplates::0.10.1912.1705-beta" && \
```

Comment:
Should this also be an `ARG` perhaps, to prevent it getting similarly out of date in the future? I know `bootstrap.ps1` should update it, so perhaps moot anyway.